src/app/video-course/video-course.module.ts

```diff
@@ -0,0 +1,9 @@
+import { NgModule } from '@angular/core'
+import { CommonModule } from '@angular/common'
+import { VideoCourseComponent } from '../components/video-course/video-course.component'
+
+@NgModule({
+  declarations: [],
+  imports: [CommonModule, VideoCourseComponent],
+})
+export class VideoCourseModule {}
```

Comment:
I think this module is redundant and can be deleted now, if you decided to take video-cource component as a standalone one